student/src/main/java/tds/student/web/handlers/TestResponseHandler.java

```diff
@@ -49,6 +50,7 @@
 
 @Controller
 @Scope ("request")
+@RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I understand that this fixes potential other methods in this class.  Just that it seems like everything doesn't actually return Json.